src/main/kotlin/nomic/domain/entities/IEntity.kt

```diff
@@ -0,0 +1,10 @@
+package nomic.domain.entities
```

Comment:
Similar idea to the IRepository file.  This is fine to leave in for now, but the extra abstraction may not really be necessary